pkg/sql/authorization_test.go

```diff
@@ -87,3 +93,74 @@ func TestCheckAnyPrivilegeForNodeUser(t *testing.T) {
 	}
 
 }
+
+// TestConcurrentGrant tests that populating MembershipCache with a high
+// priority can avoid deadlock from concurrent GRANTs, an issue detailed in
+// #117144.
+func TestConcurrentGrants(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+	skip.UnderDuress(t, "this test needs to sleep half a second")
```

Comment:
skip.UnderDuress isn't defined on this branch, so looks like we need to skip under race, stress, and deadlock explicitly.

Reply:
done